tools/custom-organization-policy-library/README.md

```diff
@@ -24,16 +24,24 @@ Configuration specific to an organization such as organization id, bundles to en
 
 Example of values.yaml
 ```
-organization: '3333334'
+organization: '11111111'
 bundles:
-  pci-dss: false
-  cis: false
+ pci-dss: false
+ cis: false
+dryrun: false
+compute:
+ computeAllowedInstanceMachineTypes:
+   params:
+     machine_types:
+     - "n2-standard-1"
+     - "n2-standard-2"
 ```
 
```

Comment:
We need to explain what organization, what is pci-dss, what is cis, what is dryrun, what is parameters and giving an exammple

Reply:
If not people will not understand how to fill it